web/src/javascript/constants.js

```diff
@@ -0,0 +1,4 @@
+export const regularColor = "#00B200";
+export const silverColor = "#ACACAC";
+export const goldColor = " #FFD700";
+export const foodMass = 20;
```

Comment:
Based on the discussions that we had today, I think this should be smaller than that, given that we'll have a start mass of ~10 for players. Do you agree?